BUILD

```diff
@@ -36,6 +36,12 @@ file_test(
     regexp = ".*docker cp .*:test_download_pkgs_at_root_packages.tar test_download_pkgs_at_root.tar.*",
 )
 
+file_test(
```

Comment:
Could you also add tests for those download_pkg targets in tests/package_managers/BUILD?

Reply:
Done

Reply:
Done